### ISSUE TITLE
Could you export WebStorage from /WebStorage.ts?

### ISSUE THREAD

**Issue by m-abs** [marcj/angular2-localStorage#48]
Hi,

I'm using _angular2-localstorage_ in a project based on https://github.com/NathanWalker/angular2-seed-advanced running on both web and mobile devices via Nativescript-angular.

Nativescript doesn't support _localStorage_ or _sessionStorage_, but provides it's own _application-settings_, which does much the same thing.

I'd like to use your decorate _WebStorage_ from _/WebStorage.ts_ directly, so I could provide my own storage-engine and make my own decorator based on yours.

For now I've worked around the problem by adding my own _localStorage_ and _sessionStorage_ to the global scope like this: 

``` typescript
import * as appSettings from 'application-settings';
if (!global.localStorage) {
  const localStorage = {
    getItem(key: string): string {
      try {
        return appSettings.getString(key);
      } catch (err) {
        console.log(JSON.stringify(err));
        throw err;
      }
    },
    setItem(key: string, value: string): void {
      try {
        if (value === null || value === undefined) {
          appSettings.remove(key);
          return;
        }

        appSettings.setString(key, value);
        return;
      } catch (err) {
        console.log(JSON.stringify(err));
        throw err;
      }
    },
  };
  global.localStorage = localStorage;
}

if (!global.sessionStorage) {
  const values = new Map<string, string>();
  const sessionStorage = {
    getItem(key: string): string {
      try {
        return values.get(key);
      } catch (err) {
        console.log(JSON.stringify(err));
        throw err;
      }
    },
    setItem(key: string, value: string) {
      try {
        if (value === null || value === undefined) {
          values.delete(key);
          return;
        }

        values.set(key, value);
        return;
      } catch (err) {
        console.log(JSON.stringify(err));
        throw err;
      }
    },
  };
  global.sessionStorage = sessionStorage;
}

export {LocalStorageService, LocalStorageSubscriber} from 'angular2-localstorage/LocalStorageEmitter';
```

Long story short: I'd like you to export _WebStorage_ and _IWebStorage_ from _/WebStorage.ts_
